src/features/Instructors/InstructorsFilters/index.jsx

```diff
@@ -20,18 +24,22 @@ const initialFilterState = {
   instructorEmail: '',
   courseSelected: null,
   inputType: 'name',
+  active: false,
```

Comment:
I have a problem when I don't enable instructors features site configuration, as by default this is false, the query is  sensed when I filtered data so only get the inactive instructors